modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -73,7 +73,7 @@ metadata:
 ----
 $ oc apply -f ./<test-bridge>.yaml <1>
```

Comment:
@ShaunaDiaz should this line be italicized?

Reply:
Looks like it, according to https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values.